src/_pytest/pythonpath.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from typing import List
+
+import pytest
```

Comment:
Let's go with `import pytest` only, i.e. not import from `_pytest` at all -- seems nicer when possible, even for internal plugins.

Reply:
I think so, as well. 
Looks like `from pytest import Config` works, for the type information.